logstash/templates/logstash/conf.d/280_filter_mongodb_log.conf

```diff
@@ -0,0 +1,45 @@
+filter {
+
+  if [type] == 'mongodb_log' {
+
+    grok {
+      patterns_dir => ["/etc/logstash/patterns"]
+      match => [
+        "message", "%{MONGODB_LOG_BASE}"
+      ]
+    }
+
+    date {
+      match => [ "mongodb_log_timestamp", "ISO8601" ]
```

Comment:
I think we should remove the date field once we've parsed it having the same value in `@timestamp` and mongodb_log_timestamp just gives us confusion I think. Something like this should do it:

Something like:

```
date {
  match => [ "mongodb_log_timestamp", "ISO8601" ]
  remove_field => [ 'mongodb_log_timestamp' ]
}
```